gcp/compute_engine/cloudrun_serverless_neg_https_lb/main.tf

```diff
@@ -0,0 +1,23 @@
+resource google_compute_region_network_endpoint_group serverless_neg {
```

Comment:
Self is our naming standard unless there is a multiple requirement which dictates otherwise